virttest/utils_libvirt/libvirt_vmxml.py

```diff
@@ -9,8 +9,18 @@
 
 from avocado.core import exceptions
 
+from virttest.libvirt_xml.devices import controller
```

Comment:
Please reorder these lines and move them after line 22?

Reply:
done